android-components/components/browser/state/src/main/java/mozilla/components/browser/state/reducer/TranslationsStateReducer.kt

```diff
@@ -66,13 +67,18 @@ internal object TranslationsStateReducer {
                 action.translationEngineState.requestedTranslationPair?.fromLanguage == null ||
                 action.translationEngineState.requestedTranslationPair?.toLanguage == null
             ) {
+                var translationsError: TranslationError? = null
+                if (action.translationEngineState.detectedLanguages?.supportedDocumentLang == false) {
+                    translationsError = TranslationError.LanguageNotSupportedError(cause = null)
+                }
```

Comment:
Could we move this below the `// In an untranslated state` comment? I think we want to see the comment from the get go when we enter this conditional block because that's what it represents.

Reply:
Thanks, good call!